tasks/config/run.js

```diff
@@ -150,7 +150,6 @@ module.exports = function (grunt) {
       flags: [
         ...funcTestServerFlags,
         '--dev',
-        '--dev_mode.enabled=false',
```

Comment:
how is this relevant ?

Reply:
This flag was actually used to disable the `src/core_plugins/dev_mode` plugin, during functional tests and for nothing else. I removed that plugin completely, since it only hold one spy mode that is now gone, so the flag is no longer needed.